libs/designsystem/src/lib/directives/size/size.directive.ts

```diff
@@ -15,10 +16,10 @@ export class SizeDirective {
   @HostBinding('class.lg')
   isLargeSize: boolean;
   @Input() set size(size: Sizes) {
-    this.isExtraSmall = size === 'xs';
-    this.isSmallSize = size === 'sm';
-    this.isMediumSize = size === 'md';
-    this.isLargeSize = size === 'lg';
+    this.isExtraSmall = size === Sizes.extraSmall;
+    this.isSmallSize = size === Sizes.small;
+    this.isMediumSize = size === Sizes.medium;
+    this.isLargeSize = size === Sizes.large;
```

Comment:
Is related to the PR/issue at hand? If not please revert :)

Reply:
Boyscouting as I initially changed that file. I can certainly revert it, but it's a harmless change that makes the code cleaner?

Reply:
I actually agree with that 👍 
Only thing is, in the version history it will now look like this change was related to adding support for medium sized inputs, which it isn't :)

Reply:
I reverted the change